report.txt

```diff
@@ -0,0 +1,191 @@
+Targets
```

Comment:
I think we should remive this report and https://github.com/opensearch-project/dashboards-observability/pull/983/files#diff-df3e51bb2577d27a12a41e5efd1630341367497497b26a0763d5749e45dc8e42 file.